internal/provider/harness_pterraform_resource_test.go

```diff
@@ -68,6 +68,10 @@ resource "imagetest_feature" "test" {
       name = "Access and create some stuff"
       cmd = "kubectl get po -A && kubectl create ns test"
     },
+    {
+      name = "Access the new namespace"
+      cmd = "k get ns test"
+    },
```

Comment:
you want to move this into one of the `harness_k3s_resource_test.go` test cases since the change above only affects the k3s sandbox

Reply:
I did move my test but ended up having to remove the test because the alias is not available when we are not in an interactive shell.

If we did something like this, then we would have access to the `k` outside the shell.
```
#!/bin/sh
exec kubectl
```